example/test/company_test.exs

```diff
@@ -164,4 +142,13 @@ defmodule CompanyTest do
       meta: nil
     }
   end
+
+  # Company related functions
+  def company_count(), do: (from company in Company, select: count(company.id))
+  def first_company(), do: (first(Company, :id) |> preload(:people))
+  def new_company(attrs), do: Company.changeset(%Company{}, attrs)
```

Comment:
new_company and updated_company functions could be removed for more explicitness. I think Company.changeset(company, attrs) is better than update_company(company, attrs)